src/scripts/main.js

```diff
@@ -1,7 +1,91 @@
+/* eslint-disable no-shadow */
 'use strict';
 
 // Uncomment the next lines to use your game instance in the browser
-// const Game = require('../modules/Game.class');
-// const game = new Game();
+import Game from '../modules/Game.class';
 
-// Write your code here
+const size = 4;
+
+const field = document.querySelector('.game-field');
```

Comment:
```suggestion
const field = document.querySelector('tbody');
```
it prevent to lose tbody tag after click on start.
tbody closest parent for tr